mesheryctl/internal/cli/root/system/model.go

```diff
@@ -355,7 +355,7 @@ mesheryctl exp model view [model-name]
 	`,
 	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
I believe, what @leecalcote hinting is to have a PreRunE function, which can do all these checks? or rather an Args function which can check whether the args passed are correct or not.
for e.g. refer to line no: 171 or 279

Reply:
@hexxdump yeah, this can be another enhancement.

Reply:
Please do refactor this PR to leverage Cobra's built-in capabilities. Foremost, look to how other `mesheryctl` commands handle this and implement consistently here.

Reply:
@zakisk make sense?

Reply:
@zakisk checking in again...

Reply:
PR abandoned. 

@hexxdump do you think that you can carry this work forward in a separate PR?